src/services/public_http_server/handlers/get_subscribers_v0.rs

```diff
@@ -18,15 +22,34 @@ pub async fn handler(
     State(state): State<Arc<AppState>>,
     AuthedProjectId(project_id, _): AuthedProjectId,
 ) -> Result<Response, NotifyServerError> {
+    if let Some(redis) = state.redis.as_ref() {
+        get_subscribers_rate_limit(redis, &project_id, &state.clock).await?;
+    }
+
     let accounts =
         get_subscriber_accounts_by_project_id(project_id, &state.postgres, state.metrics.as_ref())
             .await
             .map_err(|e| match e {
                 sqlx::Error::RowNotFound => {
-                    NotifyServerError::BadRequest("Project not found".into())
+                    NotifyServerError::UnprocessableEntity("Project not found".into())
                 }
                 e => e.into(),
             })?;
 
     Ok((StatusCode::OK, Json(accounts)).into_response())
 }
+pub async fn get_subscribers_rate_limit(
+    redis: &Arc<Redis>,
+    project_id: &ProjectId,
+    clock: &Clock,
+) -> Result<(), RateLimitError> {
+    rate_limit::token_bucket(
+        redis,
+        format!("subscribers-v0-{project_id}"),
+        5,
+        chrono::Duration::seconds(1),
+        1,
```

Comment:
Didn't have rate limits for this endpoint at all! Adding now.

Doesn't match docs because first we say the rate limits are less than they are, get people to migrate to the new v1 endpoint, and then actually change the rate limit.